src/data/markdown/docs/01 guides/03 Results visualization/JSON.md

```diff
@@ -13,6 +13,8 @@ k6 run --out json=my_test_result.json script.js
 
 </div>
 
+## JSON format
```

Comment:
I have not really reflected on this before, but it's not optimal that the json output isn't valid (ie. contains multiple root objects), in contrast to the summary option. Should we raise an issue about this in the k6 repo?

Reply:
Yes, go ahead.